process/throttle/block.go

```diff
@@ -0,0 +1,144 @@
+package throttle
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/logger"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+var log = logger.DefaultLogger()
+
+const jumpAbovePercent = 90
```

Comment:
You can wrap this (line 14 -> 17) in a const() instead of using the const keyword for every constant declaration.

Reply:
Done